### DISCUSSION TITLE
Usb soundcard issue

### DISCUSSION THREAD

**Discussion by doobes** [opencardev/crankshaft#593] [Q&A]
Doing an install on a RPi4.  Worked through the easy issues, but now I'm stuck.

I've got the Raspberry Pi touchscreen, a Panda USB bluetooth dongle, and a UGreen USB soundcard.

I'm connecting a Pixel 3axl through a USB cord and have paired the phone with the pi.

The reddit page is not admitting new members.

Audio (both input and output) on phone calls is routed through the phone vice through AA/Crankshaft/RPi.

Any idea of what the issue is, or where I could ask the question?

thanks

Chris

**Comment by matt2005** [comment]
Does the usb soundcard output at all?

**Comment by doobes** [comment]
Yep. Media plays just fine.

No audio from the phone.

I need to try it with a different phone though.

cd
